### PR TITLE
Determinants ant species density

### PR DESCRIPTION
**Pull request by apollinebyrdy**
If you have any question do not hesitate, 
Comment for any change
Good review and thank you, 
The Group 3

### CODE REVIEW

**Comment by Alexia-Gueguen** [comment]
Thank you for the request!

Here are the suggestions for the correction : 
-Lignes 554, 556, 558, 560 : "relation between **X** and …" --> I think you want to say "between **Y** and…." Or directly put “species richness”
-Ligne 635 : chunk name “colinearity1c” --> “collinaerity1c” (missing L)
-Ligne 661 : missing L for collinearity too
-Ligne 695 and 944 : chunk name is in French

Good luck,

Guéguen Alexia

**Review by Alexia-Gueguen — COMMENTED** [review]
Lignes 554, 556, 558, 560 : "relation between **X** and …" --> I think you want to say "between **Y** and…." Or directly put “species richness”
Ligne 635 : name chunk “colinearity1c” --> “collinaerity1c” missing L
Ligne 661 : missing L for collinearity too
Ligne 695 and 944 : name of the chunk is in French

**Comment by EliaDroneau** [comment]
> Thank you for the request!
> 
> Here are the suggestions for the correction : -Lignes 554, 556, 558, 560 : "relation between **X** and …" --> I think you want to say "between **Y** and…." Or directly put “species richness” -Ligne 635 : chunk name “colinearity1c” --> “collinaerity1c” (missing L) -Ligne 661 : missing L for collinearity too -Ligne 695 and 944 : chunk name is in French
> 
> Good luck,
> 
> Guéguen Alexia

**Comment by EliaDroneau** [comment]
I agree with Alexia, maybe it will be more clear to write "species richness". It easier to understand the aim of your statistical work.

File: references_determinant_ant_diversity.bib [hunk 1]
```diff
@@ -142,3 +163,42 @@ @software{tibshirani_bootstrap_2019
...
+	number = {2},
+	journaltitle = {Methods in Ecology and Evolution},
+	shortjournal = {Methods Ecol Evol},
+	author = {Johnson P., Barry S., Ferguson H., Muller P., Schielzeth H.},
+	date = {2015-02},
+	keywords = {generalized linear mixed models, ecology, evolution, Poisson},
+}
+
+@article{Irz_et_Levi-Valensin_2016,
+	title = {Tendance nationale de l'Indice Poisson Rivière (IPR) estimée par un Modèle Additif Généralisé (GAM)},
```
Comment (EliaDroneau) [inline references_determinant_ant_diversity.bib:189]: It is in French, maybe translate it in English.

**Review by EliaDroneau — COMMENTED** [review]
Make sure everything is in English to coincide with the rest of your script.

File: references_determinant_ant_diversity.bib [hunk 2]
```diff
@@ -142,3 +163,42 @@ @software{tibshirani_bootstrap_2019
...
+	author = {Irz P., Levi-Valensin M.},
```
[the remark this answers is not in the document]
Comment (MarieEtienne) [reply references_determinant_ant_diversity.bib:190]: But this is a reference ! If the book is in french you can not translate it :)

**Comment by MarieEtienne** [comment]
@apollinebyrdy  Please solve the conflicts before asking for review as the resolution of the conflicts might add problems!

**Comment by MarieEtienne** [comment]
You should accept my PR which correct the fail check problem

**Pull request merged**